auth_otp/views.py

```
@@ -0,0 +1,357 @@
+from .models import OTPRequest, OTPRateLimit
+
+logger = logging.getLogger(__name__)
+
+
+# ====================================
+# OTP Send Endpoint
+# ====================================
+
+@api_view(['POST'])
+@permission_classes([AllowAny])
+def send_otp(request):
```

Ruff (0.12.2)

Missing return type annotation for public function `send_otp`

(ANN201)